### PR TITLE
Fixes formatting issues and adds PHPUnit support

### PR DESCRIPTION
**Pull request by ameech**

### CODE REVIEW

File: src/Wheniwork.php [hunk 1]
```diff
@@ -12,172 +12,205 @@
...
+    private $verify_ssl   = FALSE;
 
     /**
      * Create a new instance
+     *
      * @param string $api_token The user WhenIWork API token
      * @param array $options Allows you to set the `headers` and the `endpoint`
      */
-    function __construct($api_token = null, $options = [])
+    function __construct($api_token = NULL, $options = [])
```
Comment (dolfelt) [inline src/Wheniwork.php]: [PSR-2.5](http://www.php-fig.org/psr/psr-2/) says we MUST use lowercase for `true`, `false`, and `null`. Sorry. :disappointed:

**Comment by dolfelt** [comment]
These updates look great. Lowercase the keywords and we're in business! :rocket:

**Comment by ameech** [comment]
I changed all of the keywords to lower case now. :walking:

**Comment by dolfelt** [comment]
Thanks! This looks great. :trophy:

**Pull request merged**